<?xml version="1.0"?>
<launch>

  <rosparam file="$(find eurobot)/config/field_params.yaml" command="load" ns="field" />
  <rosparam file="$(find eurobot)/config/particle_filter_params.yaml" command="load" ns="main_robot/particle_filter" />
  <rosparam file="$(find eurobot)/config/main_robot_params.yaml" command="load" ns="main_robot" />

  <group ns="main_robot">

    <!-- NODES -->
    <node pkg="eurobot" name="particle_filter_node" type="particle_filter_node.py" output="screen"/>

  </group>

  <node pkg="rosbag" name="bag_play" type="play" args="$(find eurobot_navigation)/bag/calibrate1.bag --loop" />

</launch>
